index.js

```diff
@@ -84,8 +84,24 @@
     return x != null && x['@@functional/placeholder'] === true;
   };
 
-  /* istanbul ignore next */
-  var Any = function Any() {};
+  var formatters = {
+    '{}': R.identity,
+    '{ord}': R.nth(_, ['first', 'second', 'third']),
+    '{repr}': R.toString,
+    '{type}': function(type) { return /^function (\w*)/.exec(type)[1]; }
+  };
+
+  //  format :: (String, [*]) -> String
+  var format = function(template, values) {
+    var idx = -1;
+    return template.replace(/[{].*?[}]/g, function(match) {
+      return formatters[match](values[idx += 1]);
+    });
+  };
+
+  var a = {name: 'a'};
+  var b = {name: 'b'};
+  var c = {name: 'c'};
```

Comment:
At a glance doens't look to have much overlap with metadata. What are these variables?